apps/pwabuilder/src/script/pages/app-report.ts

```diff
@@ -2819,7 +2819,15 @@ export class AppReport extends LitElement {
     }
   }
 
-  handleShowingTooltip(e: CustomEvent){
+  handleShowingTooltip(e: CustomEvent, origin: string, field: string){
+    // general counter
```

Comment:
Now we just get the "origin" and the field and attach it all in one place in the report card page. This will tell us when the tooltip is opened and where from.